setup.cfg

```
@@ -41,10 +41,10 @@ exclude_lines =
 ignore_errors = true
 
 [mypy]
+follow_imports = silent
```

i think the idiomatic way to suppress errors in `_vendor/` would be with this-

```ini
[mypy-pdm._vendor.*]
ignore_errors = True
```